style.css

```diff
@@ -172,7 +172,6 @@ a {
 }
 
 .project-container {
-  display: grid;
```

Comment:
- [ ] Kindly use CSS grid to align all contents of project container. The project requirements demand this.